added docker to project

dmarmul opened the pull request.

dmarmul committed:
added docker to project

dmarmul committed:
changer roles names USER on ROLE_USER etc.
added user role again by liquibase, changed sql version and added allowPublicKeyRetrieval=true&useSSL=false to spring.datasource.url.

liliia-ponomarenko commented on src/main/resources/db/changelog/db.changelog-master.yaml:
```diff
@@ -29,3 +29,33 @@ databaseChangeLog:
       file: db/changelog/changes/21-create-orders-table.yaml
   - include:
       file: db/changelog/changes/22-create-orders_items-table.yaml
+#  - include:
+#      file: db/changelog/changes/03-create-roles-table.yaml
+#  - include:
+#      file: db/changelog/changes/05-create-user-roles-table.yaml
+#  - include:
+#      file: db/changelog/changes/06-insert-admin-to-users-table.yaml
+#  - include:
+#      file: db/changelog/changes/07-rename-name-column-in-roles-table.yaml
+#  - include:
+#      file: db/changelog/changes/08-rename-users-roles-table.yaml
+#  - include:
+#      file: db/changelog/changes/09-insert-values-to-roles-table.yaml
+#  - include:
+#      file: db/changelog/changes/12-insert-admin-to-users-roles-table.yaml
```
Let's remove the commented code. Also, make all file names consistent;)

liliia-ponomarenko commented on src/main/resources/db/changelog/changes/24-insert-book-to-books-table.yaml:
```diff
@@ -0,0 +1,23 @@
+databaseChangeLog:
+  - changeSet:
+      id: insert-book-to-books-table
+      author: mateacademystudent
```
better use your name

liliia-ponomarenko commented on src/main/resources/db/changelog/changes/003-create-roles-table.yaml:
```diff
@@ -0,0 +1,21 @@
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: name
```
It looks strange that you created a table with the wrong column name and renamed it in one PR. Can you create a table with the correct name?

liliia-ponomarenko reviewed (changes requested):
Good job! Let’s  fix a few mistakes ;)

dmarmul committed:
Fixed some mistakes with liquibase:
used my name instead of mateacademystusent
removed comments
made all file names consistent

Sanderfuck reviewed (changes requested):
you have changes that nor related with docker

dmarmul commented:
> you have changes that nor related with docker

Can you explain?

dmarmul committed:
Changes debug port in JAVA_TOOL_OPTIONS from 5005 on DEBUG_PORT from .env
